dags/dag_content/tower_metrics_content.py

```diff
@@ -4,12 +4,13 @@
 from itertools import chain
 from typing import Any
 
-import synapseclient
+from synapseclient import File
```

Comment:
fwiw, there are some downsides to importing this way. The biggest ones imo are:

* No performance benefit from doing "from ... import ...". The entire `synapseclient` module is still imported.
* Need to update import statement each time you need to use something else from `synapseclient`.
* It's less explicit in the code where a method is coming from. `File` is generic enough that it could be mistaken for something from the python standard library.

More info: 
* https://stackoverflow.com/questions/710551/use-import-module-or-from-module-import
* https://google.github.io/styleguide/pyguide.html#22-imports